cpp/src/bindings.cpp

```diff
@@ -23,6 +22,55 @@ void bindArray(py::module_& m, const std::string& className) {
             }, py::keep_alive<0, 1>());
 }
 
+array* createArray(py::list columnsValues, py::list columnsNames, py::list columnsTypes, int columnSize, int columnsNumber) {
+    array* dataframe = new array();
+    dataframe->ptr = new series[columnsNumber];
+    series* columns = new series[columnsNumber];
+    for (int indice = 0 ; indice < columnsNumber ; indice ++ ) {
+        series* column = columns + indice;
+        py::str name = (py::str) columnsNames[indice];
+        column->name = strdup((char*) const_cast<char*>(((std::string) name).c_str()));
+        column->index = 0;
+        py::int_ type = (py::int_) columnsTypes[indice];
+        int properType = (int) type;
+        column->type = properType;
+        py::array data = (py::array) columnsValues[indice];
+        if (properType == 0) {
+            std::vector<std::string> values = py::cast<std::vector<std::string>>(columnsValues[indice]);
+            column->data.ptr = pypowsybl::copyVectorStringToCharPtrPtr(values);
+
+        } else if (properType == 1) {
+            std::vector<double> values = py::cast<std::vector<double>>(columnsValues[indice]);
+            column->data.ptr = pypowsybl::copyVectorDouble(values);
+        } else if (properType == 2 || properType == 3) {
+            std::vector<int> values = py::cast<std::vector<int>>(columnsValues[indice]);
+            column->data.ptr = pypowsybl::copyVectorInt(values);
+        }
+    }
+    dataframe->ptr = columns;
+    return dataframe;
+}
+
+void updateNetworkElementsWithSeries(pypowsybl::JavaHandle network, py::list columnsValues, py::list columnsNames, py::list columnsTypes, py::list indexList, int columnSize, int columnsNumber, element_type elementType, int indexCount) {
```

Comment:
Actually we could simplify a little the code by using c++ type here instead of py::list, when we know exactly what is expected:
- columnsNames and indexList --> `const std::vector<std::string>&`
- columnsTypes --> `const std::vector<int>&`

Only for the values it's necessary to use a `py::list` because values in it are heterogeneous